ldap/servers/plugins/sync/sync_persist.c

```diff
@@ -6,8 +6,8 @@
  * See LICENSE for details.
  * END COPYRIGHT BLOCK **/
 
-#include <nspr4/prlog.h>
-#include <bits/stdint-intn.h>
+#include <nspr/prlog.h>
+#include <bits/stdint.h>
```

Comment:
This is not present on glibc-based system. Perhaps it should be ifdefed?

Reply:
i also tested, **bits/stdint-intn.h** is not neccessary. i will try build at centos and ubuntu to verift not to broke anything.

then i will update pr.

the header **bits/stdint-intn.h** is very interesting. it's at ubuntu as **/usr/include/x86_64-linux-gnu/bits/stdint-intn.h** and referenced from **/usr/include/stdint.h**

Reply:
i deleted include